booker.pl

```diff
@@ -0,0 +1,267 @@
+#!/bin/perl
```

Comment:
Same here, probably this part can be reverted (?)

Reply:
yes, you're right